### PR TITLE
Add github and gitlab templates

### PR DESCRIPTION
**Pull request by edvgui**
# Description

Add templates for github and gitlab repos Pull/Merge requests.

### CODE REVIEW

File: {{cookiecutter.module_name|slugify}}/.github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Description
+
+* Short description here *
+
+Closes *Add ticket reference here*
+
+# Merge procedure
+
+Don't use the github built-in merge, but the process described [here](https://docs.internal.inmanta.com/topics/tasks/commiting_changes_modules.html)
```
Comment (wouterdb) [inline {{cookiecutter.module_name|slugify}}/.github/pull_request_template.md:9]: This whole file is outdated: see updated version here https://github.com/inmanta/std/blob/master/.github/pull_request_template.md

File: {{cookiecutter.module_name|slugify}}/.gitlab/merge_request_templates/default.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+- [ ] Code is clear and sufficiently documented (classes, methods and function docs, document important assumptions, link to reference materials)
+- [ ] Sufficient test cases (reproduces the bug/tests the requested feature)
+- [ ] Correct, in line with design
+- [ ] Type annotations
+- [ ] Issue is in the review column
+- [ ] MR is assigned to first reviewer
+
+# Useful references
+
+- [Committing changes to Modules](https://internal.inmanta.com/development/core/tasks/commiting-changes-modules.html#committing-changes-to-modules)
```
Comment (wouterdb) [inline {{cookiecutter.module_name|slugify}}/.gitlab/merge_request_templates/default.md:28]: I would make this link part of the checkbox items is is part of

File: {{cookiecutter.module_name|slugify}}/.gitlab/merge_request_templates/default.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+- [ ] reviewer 1
+- [ ] @bart or @wouter
+
+# Self Check:
+
+Strike through any lines that are not applicable (`~~line~~`) then check the box
+
+- [ ] Issue is referenced by this merge request
+- [ ] Changelog entry
+- [ ] Dev version bump has been made (if applicable) using `inmanta module release` command
```
Comment (wouterdb) [inline {{cookiecutter.module_name|slugify}}/.gitlab/merge_request_templates/default.md:18]:
```suggestion
- [ ] [Changelog entry and version bump](https://internal.inmanta.com/development/core/tasks/commiting-changes-modules.html) `inmanta module release --dev [--major|--minor|--patch] [--changelog-message "<your_changelog_message>"]`

```

File: {{cookiecutter.module_name|slugify}}/.gitlab/merge_request_templates/default.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+# Reviewers list 
+
+- [ ] reviewer 1
+- [ ] @bart or @wouter
+
+# Self Check:
+
+Strike through any lines that are not applicable (`~~line~~`) then check the box
+
+- [ ] Issue is referenced by this merge request
```
Comment (wouterdb) [inline {{cookiecutter.module_name|slugify}}/.gitlab/merge_request_templates/default.md:16]:
```suggestion
```

this is already part of the template above

File: {{cookiecutter.module_name|slugify}}/.gitlab/merge_request_templates/default.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+- [ ] Changelog entry
+- [ ] Dev version bump has been made (if applicable) using `inmanta module release` command
+- [ ] Code is clear and sufficiently documented (classes, methods and function docs, document important assumptions, link to reference materials)
+- [ ] Sufficient test cases (reproduces the bug/tests the requested feature)
+- [ ] Correct, in line with design
+- [ ] Type annotations
+- [ ] Issue is in the review column
+- [ ] MR is assigned to first reviewer
+
+# Useful references
```
Comment (wouterdb) [inline {{cookiecutter.module_name|slugify}}/.gitlab/merge_request_templates/default.md:26]: I think this section is too much, we have to keep this minimal

**Review by wouterdb — CHANGES_REQUESTED** [review]
I'm a bit in the doubt about including references to internal stuff in public repo's, but I guess it is ok

File: {{cookiecutter.module_name|slugify}}/.github/pull_request_template.md (hunk 1 above)
Comment (edvgui) [reply {{cookiecutter.module_name|slugify}}/.github/pull_request_template.md]: Indeed, lsm needs to be updated

**Review by wouterdb — APPROVED** [review]

**Pull request merged**